composer.lock

```diff
@@ -5648,6 +5587,67 @@
             ],
             "time": "2020-01-29T20:22:20+00:00"
         },
+        {
```

Comment:
Bootstrap theme should be removed. Uninstall the Bootstrap theme if you haven't yet done so (on Appearance page) and use `composer remove drupal/bootstrap`.

Reply:
And please don't forget to export your configuration to enable your new theme.